packages/node_modules/@ciscospark/spark-core/src/lib/credentials/token.js

```diff
@@ -269,6 +269,7 @@ const Token = SparkPlugin.extend({
     return this.spark.request({
       method: 'POST',
       uri: this.config.tokenUrl,
+      addAuthHeader: false,
```

Comment:
false should be the default so this isn't necessary

Reply:
This is necessary if we make the value as true . We still should make this as false for token request